ravencoin/client.go

```diff
@@ -56,28 +56,28 @@ const (
 type requestMethod string
 
 const (
-	// https://bitcoin.org/en/developer-reference#getblock
+	// https://ravencoin.org/en/developer-reference#getblock
 	requestMethodGetBlock requestMethod = "getblock"
 
-	// https://bitcoin.org/en/developer-reference#getblockhash
+	// https://ravencoin.org/en/developer-reference#getblockhash
 	requestMethodGetBlockHash requestMethod = "getblockhash"
 
-	// https://bitcoin.org/en/developer-reference#getblockchaininfo
+	// https://ravencoin.org/en/developer-reference#getblockchaininfo
 	requestMethodGetBlockchainInfo requestMethod = "getblockchaininfo"
 
-	// https://developer.bitcoin.org/reference/rpc/getpeerinfo.html
+	// https://developer.ravencoin.org/reference/rpc/getpeerinfo.html
 	requestMethodGetPeerInfo requestMethod = "getpeerinfo"
 
-	// https://developer.bitcoin.org/reference/rpc/pruneblockchain.html
+	// https://developer.ravencoin.org/reference/rpc/pruneblockchain.html
 	requestMethodPruneBlockchain requestMethod = "pruneblockchain"
 
-	// https://developer.bitcoin.org/reference/rpc/sendrawtransaction.html
+	// https://developer.ravencoin.org/reference/rpc/sendrawtransaction.html
 	requestMethodSendRawTransaction requestMethod = "sendrawtransaction"
 
-	// https://developer.bitcoin.org/reference/rpc/estimatesmartfee.html
+	// https://developer.ravencoin.org/reference/rpc/estimatesmartfee.html
 	requestMethodEstimateSmartFee requestMethod = "estimatesmartfee"
 
-	// https://developer.bitcoin.org/reference/rpc/getrawmempool.html
+	// https://developer.ravencoin.org/reference/rpc/getrawmempool.html
```

Comment:
Every [ravencoin.org](url) link is 404, I think you should leave the links to the bitcoin doc since this is what it's based on anyway.